<p>
This rule raises an issue whenever if finds a FIND statement without the NO-ERROR option, as AVM may raise a run-time error when the requested record is not found.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  FIND FIRST Customer NO-LOCK.
</pre>

<h2>Compliant Solution</h2>
<pre>
  FIND FIRST Customer NO-LOCK NO-ERROR.
</pre>
