<p>
This rule raises an issue whenever if finds a LENGTH / OVERLAY /SUBSTRING function call which doesn't provide the 'type' parameter.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  /* Will display 1 */
  MESSAGE LENGTH('&#x4e57;').
</pre>

<h2>Compliant Solution</h2>
<pre>
  MESSAGE LENGTH('&#x4e57;', 'CHARACTER'). /* Only one character */
  MESSAGE LENGTH('&#x4e57;', 'COLUMN').    /* Which takes 2 columns to display */
  MESSAGE LENGTH('&#x4e57;', 'RAW').       /* And 3 bytes in UTF-8 */
</pre>
