<p>When NEXT or LEAVE is used without a block label, it is not clear which loop will be exited</p>

<h2>Noncompliant Code Example</h2>
<pre>
  DO WHILE TRUE :
     FOR EACH customer :
        FOR EACH order OF customer :
            IF order.shipped THEN DO:
               DISPLAY order.shippingdate.
               NEXT. /* Which block will be used ? */ 
            END.
        END.
     END.
  END.
</pre>

<h2>Compliant Solution</h2>
<pre>
  audit_salesrep:
  FOR EACH SalesRep :
     FOR EACH customer OF SalesRep :
        FOR EACH Order OF Customer :
            IF order.shipped THEN DO:
               DISPLAY order.shippingdate.
               NEXT audit_salesrep.
            END.
        END.
     END.
  END.
</pre>
