<p>Compilation failure on UNIX systems when a backslash is used in include directive</p>

<h2>Noncompliant Code Example</h2>
<pre>
 { path\to\file.i }
</pre>

<h2>Compliant Solution</h2>
<pre>
 { path/to/file.i }
</pre>
