<p>
This rule raises an issue whenever it finds a variable, temp-table, property or parameter definition without the NO-UNDO attribute.
Any object without this attribute will keep a before-image copy in transactions, leading to I/O in the LBI file.
While in some cases you may want to use the UNDO feature on variables, it's in most cases useless.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  DEFINE VARIABLE myVar AS CHARACTER.
</pre>

<h2>Compliant Solution</h2>
<pre>
  DEFINE VARIABLE myVar AS CHARACTER NO-UNDO.
</pre>
