<p>
This rule raises an issue whenever a quoted string does not specify the :U or :T attribute.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  ASSIGN var1 = 'test'
         var2 = 'another value'.
</pre>

<h2>Compliant Solution</h2>
<pre>
  ASSIGN var1 = 'test':U
         var2 = 'another value':T.
</pre>
