<p>Case of class name has to match case of file name. Different case can lead to various compilation and runtime problems on UNIX.</p>

<h2>Noncompliant Code Example</h2>
<pre>
  MyClass.cls
  ***********

  PUBLIC CLASS myclass:
  
  END CLASS.
</pre>

<h2>Compliant Solution</h2>
<pre>
  MyClass.cls
  ***********

  PUBLIC CLASS MyClass:
  
  END CLASS.
</pre>
